fury/utils.py

```diff
@@ -964,3 +964,14 @@ def check_order(vert, triarr):
         else:
             correct_vert[nb] = i
     return correct_vert
+
+def hex_color(color):
```

Comment:
Can you rename this function `hex2rgb` or `hex_to_rgb`?
Can you add docstring (documentation for this function? Look at the functions above